src/components/Content/Content.component.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { items } from '../../mocks/youtube-videos-mock.json';
+
+import { ContentWrapper, CardContainer, Card, CardThumbnail, CardContent } from './Content.styles';
+
+const Content = () => {

```

Comment:
I would change the component name for something more descriptive and to make it more reusable I would pass the data as a prop.